spec/services/hyrax/resource_characterizer_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Hyrax::ResourceCharacterizer do
+  subject(:characterizer) { described_class.new(source: resource) }
+
+  before do
+    allow(Hyrax::FileSet).to receive(:find).with(resource.id).and_return(resource)
+    allow(Hydra::Works::CharacterizationService).to receive(:run)
+    allow(Hyrax.persister).to receive(:save)
+    allow(CreateDerivativesJob).to receive(:perform_later)
```

Comment:
is this just to avoid running the job? if so, it shouldn't be needed since we use the test queue adapter by default in the test suite.